src/client/components/Autocomplete/AutocompleteInput.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
```

Comment:
Added child component to make code more readable
Removed our TextInput because it wouldn't work correctly even with object spreading (class TextInput marked as refactor component in backlog anyway)